mappings/net/minecraft/block/MetalBlock.mapping

```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/class_55 net/minecraft/block/MetalBlock
```

Comment:
so we're calling diamonds a metal?

Reply:
yes

Reply:
it makes sense
MaterialBlock got me confused when I was mapping Material